apps/docs/src/development/packages/changelogs/[name].md

```diff
@@ -35,6 +35,12 @@ const { params } = useData();
 
 </div>
 
+<div v-else-if="params.name === 'icons'">
```

Comment:
How can this site be reached? The Icons package is not listed under packages as the other ones.

<img width="269" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/1d18c33c-0b59-4c96-b049-2eb0b71a8f2c">

\+ http://localhost:5173/development/packages/changelogs/icons.html is empty, is it because it is new and we have not published yet? :)

Reply:
found it in apps/docs/src/resources/icons.md, but shouldn't there be a packages entry as well, if only to link to that page? if people are coming from the npmjs perspective to the docs. :)

Reply:
Yes its empty because it has not been published yet. I added an entry for the icons and linked to the resources page